### DISCUSSION TITLE
Are there plans for enabling GitHub Sponsors

### DISCUSSION THREAD

**Discussion by tomkerkhove** [PrometheusClientNet/Prometheus.Client#143] [Q&A] [answered]
Are there plans for enabling GitHub Sponsors instead of external platforms?

I'm happy to sponsor a small amount to the organization.

**Comment by phnx47** [comment]
@tomkerkhove  Hi! I have some issues with register in GitHub Sponsors. My country doesn't support it yet. But later, I will do this from a European address.

**Reply by phnx47** [reply]
@sanych-sun Oh, you in Canada, maybe you can do this?

**Comment by phnx47** [comment, marked as answer]
@tomkerkhove  I did it! https://github.com/sponsors/phnx47 . I opened personal, but I will share any income with @sanych-sun that will grant for prom-client-net

**Reply by tomkerkhove** [reply]
Great to hear, just so you know - You can also enable it on an organization.

**Reply by phnx47** [reply]
I tried but issues with verify, I can't enable it right now

**Reply by sanych-sun** [reply]
That's OK with me if all donations will deposit directly to Sergey. At least for now ;)
